### PR TITLE
Intelligent agents in ai

### LINKED ISSUE #695: Intelligent Agents in Artificial Intelligence
### Brief Summary: 
An intelligent agent is an entity that makes decision, which enables artificial intelligence to be put into action. This article introduces readers to various aspects regarding intelligent agents in AI. It will help in expanding their knowledge in how AI works.

### Key Takeaways: 
1. What is an intelligent agent in artificial intelligence?
2. Characteristics of intelligent agents
3. The structure of AI agents
4. Categories of intelligent agents
5. Applications of intelligent agents

### References: 
https://www.javatpoint.com/agents-in-ai

### PR DESCRIPTION
**Pull request by mbaabuones**
#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.

**NOTE: (1st time contributors - submit ONLY one article to get familiar with the process and style-once published, you can then submit any number of articles.)**
      
#### Grammar Checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical Checks
- [ ] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution Guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**

This fixes #695 .

### CODE REVIEW

**Comment by hectorkambow** [comment]
👋 Thank you for submitting this PR - we will review it ASAP - all PRs go into our queue will be looked over before being approved to be published.

**Comment by hectorkambow** [comment]
Plagiarism check done

**Comment by adrianmurage** [comment]
@mbaabuones @hectorkambow 
Shouldn't we solve this merge conflict before the review?

**Review by adrianmurage — APPROVED** [review]
@mbaabuones great job with this. I was able to understand the ideas as they were well articulated.
@lalith1403 
Could you take a look at this, see if you could add anything to it before we mark it as ready for publishing

File: articles/intelligent-agents-in-ai/index.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+  
+- **Effectors:** The environment is affected by effectors. Examples include legs, fingers, wheels, display screen, and arms.
+  
+The following diagram shows how these components are positioned in the AI system.
+
+![AI Sytem](/engineering-education/intelligent-agents-in-artificial-intelligence/ai-system.png)
+
+[Image Source: JavatPoint](https://static.javatpoint.com/tutorial/ai/images/agents-in-ai.png)
+
+Inputs (percepts) from the environment are received by the intelligent agent through sensors. This agent uses artificial intelligence to make decisions using the acquired information/ observations. Actions are then triggered through actuators.  Future decisions will be influenced by percept history and past actions.
```
Comment (lalith1403) [inline articles/intelligent-agents-in-ai/index.md:65]: Remove additional spacing

**Review by lalith1403 — APPROVED** [review]
Well written article. Technically sound and covers the topic well.

**Comment by adrianmurage** [comment]
Thank you @lalith1403 
@mbaabuones a hero image for this is needed. Kindly add one.
Great job with the article.
Ping me when you've added the hero image and I'll label the article ready for final review.

**Comment by lalith1403** [comment]
A duplicate PR is needed. This branch has merge conflicts.

**Comment by mbaabuones** [comment]
> 
> 
> Thank you @lalith1403
> @mbaabuones a hero image for this is needed. Kindly add one.
> Great job with the article.
> Ping me when you've added the hero image and I'll label the article ready for final review.

@adrianmurage Thank you. I have added the hero image.

**Comment by adrianmurage** [comment]
@hectorkambow 
This article is ready for final review. Kindly take a look, thanks.

**Comment by hectorkambow** [comment]
> @mbaabuones @hectorkambow
> Shouldn't we solve this merge conflict before the review?

@adrianmurage  Thank you yes - I didn't see it yesterday. 

@mbaabuones  I believe many of your PRs have merge conflicts - please resolve this - and moving forward make sure each article has its own branch. 

image

Image taken from our [FAQ](https://github.com/section-io/engineering-education/blob/master/new_contributors/FAQ.md) @adrianmurage

**Comment by hectorkambow** [comment]
Please transfer files to a new fresh PR with a new branch. (if the rebase does not work - we can close this PR) open a new one

**Comment by mbaabuones** [comment]
> > @mbaabuones @hectorkambow
> > Shouldn't we solve this merge conflict before the review?
> 
> @adrianmurage Thank you yes - I didn't see it yesterday.
> 
> @mbaabuones I believe many of your PRs have merge conflicts - please resolve this - and moving forward make sure each article has its own branch.
> 
> image
> 
> Image taken from our [FAQ](https://github.com/section-io/engineering-education/blob/master/new_contributors/FAQ.md) @adrianmurage

I apologize for this. I think this is happening because I didn't open the PR for 'forward and backward chaining in ai' from a new branch. After consulting you, all my new PRs have been made from new branches. I appreciate it.

**Comment by mbaabuones** [comment]
> Please transfer files to a new fresh PR with a new branch. (if the rebase does not work - we can close this PR) open a new one

I have opened a fresh PR as requested. I have used a new branch.

**Pull request closed without merging**